routes/userRouter.js

```diff
@@ -56,6 +56,18 @@ router.post('/updateUser', function (req, res) {
 });
 
 /**
+ * GET route for getting the user list of a team
+ */
+router.get('/getUserList/:teamId', function (req, res) {
+    const teamId = req.params.teamId;
+
+    getUsersFromTeam(teamId).then((result) => {
+        res.status(200).send(result.rows);
+    }).catch(() => {
+        res.status(500).send({message: "There was an error getting the user list! Please try again later."});
+    });
+});
+
  * Updates the information of a user in the database
  * This method is generic and can be used to update any field of the user
  * @param formData the data of the user
```

Comment:
Tests are breaking here because you have text without a comment block. Either comment it out or remove it since it seems to be unecessary.